### ISSUE TITLE
Do not remove "unused" serialVersionUID

### ISSUE THREAD

**Issue by cal101** [walkmod/walkmod-pmd-plugin#4]
```
public class CacheStatistics implements Serializable {
-    private static final long serialVersionUID = 0x1e28782b066ab988L;
```

**Comment by rpau** [comment]
Ok

**Comment by rpau** [comment]
Fixed at [cfe7e06](https://github.com/walkmod/walkmod-pmd-plugin/commit/cfe7e06a237fe2d21a57af5cd891dcae6fd75ded)
